src/app/modules/color-scheme/color-picker-launcher/color-picker-launcher.component.ts

```diff
@@ -21,4 +21,8 @@ export class ColorPickerLauncherComponent {
    * Output that passes along changes made to the layer's properties from the color picker
    */
   @Output() layerChange = new EventEmitter<ImageViewerLayer>();
+
+  get className(): string {
+    return `color-icon ${this.layer.label} ${this.layer.colorScheme.type === 'discrete' ? 'circle' : ''}`;
```

Comment:
Why does the label need to be added to the class list? Also it is not html safe!